frontend/pages/hosts/details/cards/HostSummary/helpers.tsx

```diff
@@ -30,13 +30,13 @@ export const DEVICE_STATUS_TAGS: DeviceStatusTagConfig = {
   unlocking: {
     title: "UNLOCK PENDING",
     tagType: "warning",
-    generateTooltip: () =>
+    generateTooltip: (platform) =>
```

Comment:
for a minute I was confused, but the fix is to make this required so the entry with the key `locked` in this dictionary evaluates to the right tooltip content 😅